vs/windows.undocked.props

```diff
@@ -102,6 +102,9 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
     <Keyword>Win32Proj</Keyword>
+    <UndockedPlatConfig Condition="'$(UndockedPlatConfig)' == ''">$(Platform)_$(Configuration)</UndockedPlatConfig>
+    <_IntDir>$(UndockedOut)obj\$(UndockedPlatConfig)\$(MSBuildProjectName)\</_IntDir>
```

Comment:
What's the difference between `MSBuildProjectName` and `ProjectName`?

Reply:
one doesn't exist early, and is defined by some inner props file. MSBuildProjectName is the real deal